access.c

```diff
@@ -385,8 +386,8 @@ IsUserInGroup(PSID sid, const PTOKEN_GROUPS token_groups, const WCHAR *group_nam
     {
         DWORD nread, nmax;
         LOCALGROUP_MEMBERS_INFO_0 *members = NULL;
-        err = NetLocalGroupGetMembers(NULL, group_name, 0, (LPBYTE *) &members,
-                                      MAX_PREFERRED_LENGTH, &nread, &nmax, &resume);
+        err = NetLocalGroupGetMembers(
+            NULL, group_name, 0, (LPBYTE *)&members, MAX_PREFERRED_LENGTH, &nread, &nmax, &resume);
```

Comment:
Here is an example of line break after opening "("

Reply:
Yeah, one can play around with the different penalty values like `PenaltyBreakBeforeFirstCallParameter` to influence this. And obviously the `ColumnLimit` plays a huge role here.